custom_components/kia_uvo/Vehicle.py

```diff
@@ -168,11 +168,15 @@ async def start_climate(self, set_temp, duration, defrost, climate, heating):
                 climate,
                 heating,
             )
+        # call update to see climate started
         self.force_update_try_count = 0
         self.force_update_try_caller = async_call_later(
             self.hass, START_FORCE_UPDATE_AFTER_COMMAND, self.force_update_loop
         )
 
+        # call update to see climate finished
```

Comment:
I am fine with this implementation, I would just increase the buffer (+1), keep a reference to async_call_later, in case there is a new `start_climate` call, so I can cancel the previous schedule and schedule a new one.

A different solution could be `set a milestone in the future if it is still ON, if yes, call force_update`. not a blocker though.

Reply:
updated buffer and added reference, waiting to merge until duration issue in USA addressed